src/WebJobs.Extensions.CosmosDB/Trigger/CosmosDBTriggerBinding.cs

```diff
@@ -96,31 +96,20 @@ public ParameterDescriptor ToParameterDescriptor()
             {
                 Name = _parameter.Name,
                 Type = CosmosDBTriggerConstants.TriggerName,
-                CollectionName = this._documentCollectionLocation.CollectionName
+                CollectionName = this._monitoredContainer.Id
             };
         }
 
-        internal static bool TryAndConvertToDocumentList(object value, out IReadOnlyList<Document> documents)
+        private IReadOnlyDictionary<string, object> GetBindingData(object value)
         {
-            documents = null;
-
-            try
-            {
-                if (value is IReadOnlyList<Document> docs)
-                {
-                    documents = docs;
-                }
-                else if (value is string stringVal)
-                {
-                    documents = JsonConvert.DeserializeObject<IReadOnlyList<Document>>(stringVal);
-                }
-
-                return documents != null;
-            }
-            catch
+            if (value == null)
             {
-                return false;
+                throw new ArgumentNullException(nameof(value));
             }
+
+            Dictionary<string, object> bindingData = new Dictionary<string, object>(StringComparer.OrdinalIgnoreCase);
+            bindingData.Add("CosmosDBTrigger", value);
```

Comment:
Was this supported before? Not sure if we need this?

Reply:
I thought it was required, but I removed it and still worked.